files/image_config/caclmgrd/caclmgrd

```diff
@@ -550,23 +550,24 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
             namespace = redisSelectObj.getDbConnector().getNamespace()
             # Pop data of both Subscriber Table object of namespace that got config db acl table event
             for table in config_db_subscriber_table_map[namespace]:
-                (key, op, fvp) = table.pop()
-                # Pop of table that does not have data
-                if key == '':
-                    continue
-                # ACL Table notification. We will take Control Plane ACTION for any ACL Table Event
-                # This can be optimize further but we should not have many acl table set/del events in normal
-                # scenario
-                elif acl_rule_table_seprator not in key:
-                    ctrl_plane_acl_notification = True
-                # Check ACL Rule notification and make sure Rule point to ACL Table which is Controlplane
-                else:
-                    acl_table = key.split(acl_rule_table_seprator)[0]
-                    if self.config_db_map[namespace].get_table(self.ACL_TABLE)[acl_table]["type"] == self.ACL_TABLE_TYPE_CTRLPLANE:
-                        ctrl_plane_acl_notification = True
-
-            # Update the Control Plane ACL of the namespace that got config db acl table/rule event
-            if ctrl_plane_acl_notification:
+                while True:
+                    (key, op, fvp) = table.pop()
+                    # Pop of table that does not have data so break
+                    if key == '':
+                        break
+                    # ACL Table notification. We will take Control Plane ACTION for any ACL Table Event
+                    # This can be optimize further but we should not have many acl table set/del events in normal
+                    # scenario
+                    if acl_rule_table_seprator not in key:
+                        ctrl_plane_acl_notification.add(namespace)
+                    # Check ACL Rule notification and make sure Rule point to ACL Table which is Controlplane
+                    else:
+                        acl_table = key.split(acl_rule_table_seprator)[0]
+                        if self.config_db_map[namespace].get_table(self.ACL_TABLE)[acl_table]["type"]  == self.ACL_TABLE_TYPE_CTRLPLANE:
```

Comment:
>  [](start = 102, length = 1)

Use only one space

Reply:
@ Done

---
In reply to: [504353871](https://github.com/Azure/sonic-buildimage/pull/5621#discussion_r504353871) [](ancestors = 504353871)